test/python/transpiler/test_calibrationbuilder.py

```diff
@@ -88,6 +230,93 @@ def compute_stretch_width(self, play_gaussian_square_pulse, theta):
         target_area = abs(theta) / self.__angle * full_area
         return max(0, target_area - risefall_area)
 
+    def test_native_cr(self):
```

Comment:
This must be a part of `TestRZXCalibrationBuilder` because this class is a superclass of two Rzx passes of echo and non-echo. You need to write tests for 
- native cr with echo
- backward cr with echo
- native cr with non-echo (backward for non echo is not supported)

with both FakeHanoi, FakeHanoiV2 and Sherbrooke.